eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteUtils.scala

```diff
@@ -60,6 +60,12 @@ object SqliteUtils {
     res.getInt("version")
   }
 
+  def updateVersion(statement: Statement, db_name: String, version: Int): Int = {
```

Comment:
```scala
  /**
    * Updates the version for a particular logical database, it will overwrite the previous version.
    * @param statement
    * @param db_name
    * @param newVersion
    * @return
    */
  def setVersion(statement: Statement, db_name: String, newVersion: Int) = {
    statement.executeUpdate("CREATE TABLE IF NOT EXISTS versions (db_name TEXT NOT NULL PRIMARY KEY, version INTEGER NOT NULL)")
    // overwrite the existing version
    statement.executeUpdate(s"UPDATE versions SET version=$newVersion WHERE db_name='$db_name'")
  }
```